modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -75,13 +75,6 @@
             v-model="checked"
             :options="treeOptions"
             @update="update"
-            @edit="openEditor"
```

Comment:
The edit event is a few levels deep and I don't think used anywhere in that thread.

Reply:
> On a related note, the AposTree component is emitting `change` in `checkProxy`, but this isn't listening for it.

Yeah I am not sure what's up with that, you'd think it would emit `input`, but going to leave that alone in this PR.

Reply:
It was wired via v-model custom event name configuration.

Reply:
ah, right